### ISSUE TITLE
improvement request: com.google.android.gms.vision.text should return the confidence level

### ISSUE THREAD

**Issue by javalc6** [googlesamples/android-vision#187]
The google OCR vision text API do not return the confidence level related to the recognized textblock.
This information would be useful to drop textblocks with low confidence level.

An issue I found is that sometimes characters like £ & ( ) and others are wrongly recognized as letters or numbers...

**Comment by pranaysharma** [comment]
Yeah confidence is greatly required as there are frequent errors in detection. Also at a character level information  about what other characters with lower confidence were detected would be great.

**Comment by YedidyahD** [comment]
I just started using Google's Vision library and really like the results.
However, I was also looking for some kind of confidence level/score. Is there anything available yet?
Any news at all?
